src/styles/createBreakpoints.spec.js

```diff
@@ -21,8 +21,20 @@ describe('createBreakpoints', () => {
   });
 
   describe('down', () => {
-    it('should work', () => {
-      assert.strictEqual(breakpoints.down('md'), '@media (max-width:959.95px)');
+    it('should work for xs', () => {
+      assert.strictEqual(breakpoints.down('xs'), '@media (max-width:599.95px)');
+    });
+
+    it('should work for md', () => {
+      assert.strictEqual(breakpoints.down('md'), '@media (max-width:1279.95px)');
+    });
+
+    it('should apply to no sizes if unknown size is specified', () => {
+      assert.strictEqual(breakpoints.down('xxl'), '@media (max-width:-0.05px)');
```

Comment:
Humm. Do we have a test with a number as first argument? I'm not sure we want to test `xxl` 😁 .